<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
   <item>
    <widget class="QDockWidget" name="dockWidget">
     <widget class="QWidget" name="dockWidgetContents">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListView" name="listViewFonctions"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutFonctionDock">
         <item>
          <widget class="QLabel" name="labelFonctionDock">
           <property name="text">
            <string>Fonction :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="fonctionLineEditDock"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutDockButtons">
         <item>
          <widget class="QPushButton" name="enregistrerPushButton">
           <property name="text">
            <string>Enregistrer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ajouterPushButton">
           <property name="text">
            <string>Ajouter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="supprimerPushButton">
           <property name="text">
            <string>Supprimer</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
